python/lsst/ap/association/ephemerisQuery.py

```diff
@@ -0,0 +1,247 @@
+# This file is part of ap_association.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Solar System Object Query to Skybot in place of the EphemerisChache task and
+internal Rubin data.
+
+Will compute the location for of SSObjects for a know visit. Currently uses
+a full external service across the web to fill data.
+"""
+
+__all__ = ["EphemerisQueryConfig", "EphemerisQueryTask"]
+
+
+from hashlib import blake2b
+import numpy as np
+import pandas as pd
+import requests
+from io import StringIO
+
+from lsst.daf.base import DateTime
+import lsst.pex.config as pexConfig
+import lsst.pipe.base as pipeBase
+
+from lsst.pipe.base import PipelineTask, PipelineTaskConfig, PipelineTaskConnections
+import lsst.pipe.base.connectionTypes as connTypes
+
+# Enforce an error for unsafe column/array value setting in pandas.
+pd.options.mode.chained_assignment = 'raise'
+
+
+class EphemerisQueryConnections(PipelineTaskConnections,
+                                dimensions=("instrument",
+                                            "visit")):
+    visitInfos = connTypes.Input(
+        doc="Information defining the visit on a per detector basis.",
```

Comment:
There is currently no way to just get visit information without retrieving it from some sort of exposure/raw object. Since the quantum is Visit as is the expected future behavior for SSO retrieval, not putting multi here causes a crash in quantum graph creation as the input is ambiguous.